### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by ShingiraiBhengesa**
In this branch, I added more media queries and made some changes for desktop responsiveness.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -447,16 +447,98 @@ footer > a {
...
 .about-myself {
   margin-top: 2em;
-  display: flex;
-  flex-direction: column;
-  justify-content: center;
-  align-items: center;
 
   /* prettier-ignore */
-  background: url(./assets/illustration_1-about_me-mobile.svg) no-repeat 100% 0%, url(./assets/illustration_2-about_me_mobile.svg) no-repeat 0 300px;
+  background: url(./assest/illustration_1-about_me-mobile.svg) no-repeat 100% 0%, url(./assest/illustration_2-about_me_mobile.svg) no-repeat 0 300px;
```
Comment (keroloussamy) [inline style.css:541]: - [ ] Kindly make these two images appear on the mobile version too, you have to keep the mobile version working also matching the Figma design.

| Your design  | Figma design |
| ------------- | ------------- |
| image | image |

File: style.css [hunk 2]
```diff
@@ -505,6 +587,22 @@ footer > a {
   align-items: center;
 }
 
+@media (min-width: 1024px) {
+  /* prettier-ignore */
+  .about-myself {
+    background: url(./assets/about_me_desktop_left.svg) no-repeat 0% 20%, url(./assets/about_me_desktop_rigth.svg) no-repeat 100% 0%;
+  }
+}
```
Comment (keroloussamy) [inline style.css:595]: - [ ] Kindly you can add `background-size: 13vw;` here to make the image not come over the text.

image

**Review by keroloussamy — CHANGES_REQUESTED** [review]
Hi @ShingiraiBhengesa, 

Good job so far! You did really great work
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Check the comments under the review._

- [ ] x

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by medaminedev66 — APPROVED** [review]
## Hi @ShingiraiBhengesa, 

# Status: Approved ✔️
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚢
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**